src/system_monitor_pg.erl

```diff
@@ -0,0 +1,264 @@
+-module(system_monitor_pg).
+
+-behaviour(gen_server).
+-export([ init/1
+        , handle_continue/2
+        , handle_call/3
+        , handle_info/2
+        , handle_cast/2
+        , format_status/2
+        , terminate/2
+        ]).
+
+-behaviour(system_monitor_callback).
+-export([ start/0, stop/0, produce/2 ]).
+
+-include_lib("system_monitor/include/system_monitor.hrl").
+-include_lib("hut/include/hut.hrl").
+
+-define(SERVER, system_monitor_pg).
+-define(FIVE_SECONDS, 5000).
+-define(ONE_HOUR, 60*60*1000).
+
+%%%_* API =================================================================
+start() ->
+  {ok, _Pid} = gen_server:start_link({local, ?SERVER}, ?MODULE, [], []),
```

Comment:
If I read this PR correctly, the system_monitor_pg gen_server is linked to the system_monitor gen_server, which feels like they're too closely coupled. Could system_monitor_pg instead add itself under system_monitor_sup?

Reply:
Yes it could. Change upcoming :+1: